src/anet.c

```diff
@@ -194,6 +194,7 @@ int anetKeepAlive(char *err, int fd, int interval)
     }
 
     intvl = idle/3;
+    if (intvl < 10) intvl = 10; /* kernel expects at least 10 seconds */
```

Comment:
IFAIK, if it's less than 10 sec, kernel will also force it to 10 sec.

Reply:
Really? Any references to back this up? If that's true, I wonder if this rule also applies to the maximum for `TCP_KEEPALIVE_ABORT_THRESHOLD/TCP_KEEPIDLE` and `TCP_KEEPINTVL`. If so, we may want to eliminate the previous maximum check of ten days as well.

Reply:
please ignore me, i missed on solaris.